core/src/test/java/org/infinispan/persistence/support/AsyncStoreTest.java

```diff
@@ -412,7 +415,7 @@ public void call() throws InterruptedException {
                   }
                });
 
-               Exceptions.expectException(TimeoutException.class, () -> f.get(100, TimeUnit.MILLISECONDS));
+               Exceptions.expectException(TimeoutException.class, () -> f.get(DummyInMemoryStore.SLOW_STORE_WAIT * 2 + 5, TimeUnit.MILLISECONDS));
```

Comment:
IIRC passivation actually has to hit the store twice, so it really should have been 200 originally. So the original number was too low. So I really only added 5 milliseconds. The biggest change is that it is based on the dummy slow time so you can tweak it in tests if needed.

Reply:
This test's writes would finish in <1ms if they weren't waiting for `store.delayedFuture` (unless you're debugging, and then 200ms vs 10ms doesn't change anything).

If it was using `slow(true)`, then it would make sense to give it 200ms to finish both writes, but it's not, so `f` will complete as soon as `store.delayedFuture` is completed.